AcademicManagerV1/src/com/academic/db/dao/CourseDAO.java

```diff
@@ -32,12 +32,15 @@ public CourseDAO(Connection conn) throws SQLException {
 		ResultSet.TYPE_SCROLL_INSENSITIVE, ResultSet.CONCUR_READ_ONLY);
 		
 		addStatement = dbConnection.prepareStatement(
-				"INSERT SQL ");
+				"INSERT INTO course (courseId, title, cost, description, startingDate, endingDate, isActive, isDeleted) VALUES ('?','?','?','?','?','?','?','?')");
+");
 
 		updateStatement = dbConnection.prepareStatement(
 				"UPDATE course SET title= ? , cost= ?, description= ?, startingDate= ?, endingDate= ? , isActive= ? WHERE courseId=?");
 
-		deleteStatement = dbConnection.prepareStatement("INSERT SQL");
+		deleteStatement = dbConnection.prepareStatement("UPDATE 'course' SET 'isActive'=?,'isDeleted'=? WHERE courseId=?");
```

Comment:
Remove the single quotes and the isActive = ?
In lines 26, 28, 31  remove isActive = 1 and replace with isDeleted = 0